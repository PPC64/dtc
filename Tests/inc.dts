/dts-v1/;
/include/ "inc1.dtsi"
/include/ "inc2.dtsi"


/ {
  foo {
    compatible = "simple-bus";
    /include/ "bar.dtsi"
  };
};

