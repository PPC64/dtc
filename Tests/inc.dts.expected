/dts-v1/;

/  {

	foo = "bar";
	foo {

		compatible = "simple-bus";
		foo = "bar";
	};
};
